### ISSUE TITLE
Unable to make transactions

### ISSUE THREAD

**Issue by chanukya246** [ethereum/mist#3264]
<img width="1175" alt="screen shot 2017-11-08 at 10 06 54 pm" src="https://user-images.githubusercontent.com/33529876/32630884-ea5a54a4-c563-11e7-8cbc-7d5827441e69.png">

Hi, I am unable to track my golem tokens in my main account(disappeared all of sudden). 
And this wallet is giving me so hard time to transfer the funds from here to another wallet.

My balances are [1.07xxxxx Ether] & [1835.xxx golem] tokens. 
I need a way to transfer my funds to bittrex account. Please help me out. Thanks in advance.

<img width="997" alt="screen shot 2017-11-09 at 3 33 03 pm" src="https://user-images.githubusercontent.com/33529876/32630878-e60b2522-c563-11e7-9291-69dda3966f69.png">

**Comment by btruckenmiller** [comment]
Same issue - the transaction registers in log file that I requested it, but doesn't actually get processed.

**Comment by ArcadeEngine78** [comment]
Same issue here but regardless of my "gas fee" I always get the following message:

"It seems this transaction will fail. If you submit it, it may consume all the gas you provide."

Looking to send my ETH to different exchanges and there is no way I can move it.

This was supposed to be easy right?

**Comment by chanukya246** [comment]
yes, Unless someone helps us out to fix this issue we can't use 'em anymore.

**Comment by chanukya246** [comment]
@evertonfraga Could you please help me out fixing the above issue. Thanks in advance :)

**Comment by chanukya246** [comment]
Hi Guys , I found a solution to above problem.
1] Make sure you have the keystore backup folder with UTC/json files of your Main Account or where ever u stored the Ether/coins.
2] go to https://www.myetherwallet.com/#view-wallet-info & click on keystore/json file option & upload your UTC/json file mentioned in point no. 1 & enter ur password.
3] Now u can see all ur balances of ur local mist wallet in this account. Then transfer it to ur destination wallet. follow this video https://www.youtube.com/watch?v=phht73IvUDI [from 2.20 sec]

follow this:
@btruckenmiller  @ArcadeEngine78  

Thank you @loggerhead51
